### ISSUE TITLE
Le script fonctionne ????

### ISSUE THREAD

**Issue by alexdou32** [TrainerRoshi/Roshi-HHeroes-Trainer#20]
Je vois qu'il y a pleins de mises a jours de faites mais je n'arrive pas a le faire fonctionner. SVP aidez moi.

**Comment by TrainerRoshi** [comment]
Oui, le script fonctionne. Tous les mises à jours sont faites automatiquement.
![Screenshot 2021-04-23 160319](https://user-images.githubusercontent.com/48032437/115924362-9db11f80-a44d-11eb-8801-7a1c0985cca9.png)

**Comment by TrainerRoshi** [comment]
Tampermonkey installera automatiquement le script si tu aller à:

http://github.com/TrainerRoshi/Roshi-HHeroes-Trainer/raw/master/RoshiHHeroesTrainer.user.js

Sinon, tu as un problème avec Tampermonkey ou avec d'autres scripts tu exécute en même temps. Poster une capture d'écran d'erreur.

**Comment by NOCH-454** [comment]
Bonjour Roshi :)
SUPER TAF pour les script HH !! Excellent !!
Par contre j'ai un problème avec Comix H maintenant dès la mise en route du script : 
![PBMsousCH](https://user-images.githubusercontent.com/86018595/122271309-8e939e80-cedf-11eb-814b-adab03446483.jpg)
?? pourquoi fait-il ça ??
Seul solution pour l'instant : désactiver le script pour www.comixharem.com ...
Faut-il le mettre avec des réglages par défaut ? Si oui comment faire, vu que je me fais shooter tout de suite si je l'active ... Tampermonkey peut-il mettre toutes les options du script en off, pour que je puisse après le paramétrer en validant les options désirées ??
Un grand merci encore, TOP :)

**Comment by NOCH-454** [comment]
Salut Roshi :)

J'ai compris pourquoi : l'option des "filles cachées" étaient validées quand il a installé le script... Une fois le 2ème niveau commencé, j'ai pu réactiver le script et ça fonctionne très bien :)

Conclusion : on peut avoir des pbm pendant le 1er niveau, mais cet apprentissage est rapide à passer pour débuter le 2ème monde et ne plus avoir de soucis !!

Super boulot encore !! TOP

**Comment by TrainerRoshi** [comment]
Pour mettre toutes les options du script en off:
![Screenshot (76)](https://user-images.githubusercontent.com/48032437/122603591-9586f200-d042-11eb-945c-872cce6857ff.png)

Ou faites simplement un clic droit et supprimez le Local Storage.

J'ajouterai une mesure de sécurité pour que les gens ne l'activent pas s'ils n'ont pas de troll à combattre. J'ai ajouté tant d'autres mesures de sécurité, j'ai oublié celui-là.

Je suis heureux que tu aimes mon script  (✿◠‿◠)

**Comment by TrainerRoshi** [comment]
Bien. À partir de la version 1.25, tu n'auras plus cette erreur.

**Comment by NOCH-454** [comment]
T'es l'meilleur !! Trop balaise ;-)
